### PR TITLE
fix: Update gitignore to ignore Python venv files and folders

### LINKED ISSUE #261: [BUG] .gitignore does not ignore python venv folders
### Describe the issue:

Upon cloning the repository and setting up a virtual environment in Python, the new folders are tracked in the repository, which they should not be.

I propose adding the following lines to .gitignore:

```gitignore
.Python
[Bb]in
[Ii]nclude
[Ll]ib
[Ll]ib64
[Ll]ocal
[Ss]cripts
pyvenv.cfg
.venv
pip-selfcheck.json
```

### Reproduce the code example:

In the directory above `Fast-F1`:
```bash
python3 -m venv Fast-F1
```

### Error message:

image

### PR DESCRIPTION
**Pull request by ndv99**
## Changes

Added the following files/folders to `.gitignore`:

```gitignore
.Python
[Bb]in
[Ii]nclude
[Ll]ib
[Ll]ib64
[Ll]ocal
[Ss]cripts
pyvenv.cfg
.venv
pip-selfcheck.json
```

## Fixes

Fixes #261

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,17 @@ exp*.py
 venv/
 act/
 .actrc
+.Python
+[Bb]in
+[Ii]nclude
+[Ll]ib
+[Ll]ib64
+[Ll]ocal
+[Ss]cripts
```
Comment (oscr) [inline .gitignore:19]: Should scripts be in here? That's a directory used today to store scripts in

Comment (ndv99) [reply .gitignore:19]: That's interesting actually - on Windows, if you create a virtual environment, it generates a `Scripts` directory, where the `activate` script is stored. I can add a couple of exceptions to ignore the scripts already in there if that suits?

Comment (oscr) [reply .gitignore:19]: That's an interesting to know! In other words having the `scripts` folder as is can cause issues for Windows? So in other words that directory should maybe be moved?

Comment (ndv99) [reply .gitignore:19]: Exactly. Either moved or renamed I reckon, something like `ff1_scripts` could do well

Comment (oscr) [reply .gitignore:19]: That is a good catch and should have it's own issue I think

Comment (ndv99) [reply .gitignore:19]: That's absolutely fine. I'll update this PR to not have the `Scripts` directory in the gitignore and open an issue for the other one

Comment (ndv99) [reply .gitignore]: Sure thing, I've removed it just now but I've also created an issue for that: https://github.com/theOehrly/Fast-F1/issues/265

Comment (theOehrly) [reply .gitignore]: I have to fully disagree with this PR. You should not create a venv inside the project root directory. You should use a separate folder for this. See my explanation in #261

Comment (ndv99) [reply .gitignore]: Ahh fair enough, that makes sense. I'll leave this one.

**Pull request closed without merging**